src/Foundatio.RabbitMQ/Messaging/RabbitMQMessageBus.cs

```diff
@@ -117,21 +131,22 @@ public class RabbitMQMessageBus : MessageBusBase, IMessageBus {
                 return;
             _logger.Trace("Message Publish: {messageType}", messageType.FullName);
 
-            // RabbitMQ only supports delayed messages with a third party plugin.
-            if (delay.HasValue && delay.Value > TimeSpan.Zero) {
```

Comment:
I have this logic in the PublishAsync now :

`            // if the rabbitmq plugin is not availaible then use the base class delay mechanism
            if (!_delayedExchangePluginEnabled  && delay.HasValue && delay.Value > TimeSpan.Zero) {
                _logger.Trace("Schedule delayed message: {messageType} ({delay}ms)", messageType.FullName, delay.Value.TotalMilliseconds);
                await AddDelayedMessageAsync(messageType, message, delay.Value).AnyContext();
                return;
            }`
When I run the unit test case with the plugin enabled then the test case for CanSendDelayedMessage fails because the above logic will not get executed. With plugin disabled the test case passes.